src/stores/task/actions.ts

```
@@ -1,21 +1,160 @@
+                    response &&
+                    typeof response === 'object' &&
+                    'success' in response &&
+                    'data' in response
+                );
+            }
+            if (!isApiResponse(response)) {
+                throw new Error('Invalid API response');
+            }
+
+            // Add type guard
+            function isApiResponse(response: any): response is ApiResponse {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'isApiResponse'. Consider to delete it or rename it.

'isApiResponse' is defined here:


(lint/suspicious/noRedeclare)